addons/mail/static/src/components/message/message.scss

```diff
@@ -2,38 +2,34 @@
 // Layout
 // ------------------------------------------------------------------
 
+// TODO : When the migration to BS5 for the backend AND the frontend
+// will be done, we could use BS5 classes to handle the position values
+// (eg. position-absolute top-0 end-0 mt-n3)
 .o_Message_actionListContainer {
     @include o-position-absolute($top: - map-get($spacers, 3), $right: 0);
+    z-index: 10; // Place the element in front of the Composer when they overlap
```

Comment:
This looks like the only change that seems unrelated to scope of PR.
Is it a fix?

Reply:
This fix has been moved here : https://github.com/odoo/odoo/pull/89610